genesis-info/dn.log

```diff
@@ -0,0 +1,300 @@
+17cc318de Add instruction on update system testing
```

Comment:
Why is it in `genesis-info`?

Reply:
@gromakovsky b/c it's how last generate script is implemented (devops guys)
Had neither time nor wish to refactor it